packages/table/src/table.tsx

```diff
@@ -27,6 +33,13 @@ import { Locator } from "./locator";
 import { IRegion, IStyledRegionGroup, RegionCardinality, Regions, SelectionModes } from "./regions";
 import { TableBody } from "./tableBody";
 
+export type TableLoadingOption = ColumnLoadingOption | RowHeaderLoadingOption;
+export const TableLoadingOption = {
+    CELL: "cell" as TableLoadingOption,
+    COLUMN_HEADER: "column-header" as TableLoadingOption,
+    ROW_HEADER: "row-header" as TableLoadingOption,
+};
```

Comment:
This enum basically already exists as `RegionCardinality`. I'd like to combine these concepts for DRY. In fact, this should probably work just like SelectionMode, whose type is defined as `RegionCardinality[]`